source/wp-content/themes/wporg-developer-2023/templates/single.html

```diff
@@ -1,5 +1,44 @@
 <!-- wp:template-part {"slug":"header","className":"has-display-contents"} /-->
 
-<!-- wp:pattern {"slug":"wporg-developer-2023/single-content"} /-->
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained","wideSize":"1280px","contentSize":"680px"},"style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}}} -->
```

Comment:
Can you add in your [suggestion here](https://github.com/WordPress/wporg-developer/pull/190#issuecomment-1414364527) for block spacing.